package.json

```diff
@@ -41,6 +41,7 @@
 		"@wordpress/i18n": "file:packages/i18n",
 		"@wordpress/is-shallow-equal": "file:packages/is-shallow-equal",
 		"@wordpress/keycodes": "file:packages/keycodes",
+		"@wordpress/mobile": "file:packages/mobile",
```

Comment:
I think we shouldn't add the mobile package here to avoid impacting the web a lot. I consider this root package.json as the package.json for the web application. It can also be moved to packages/web (but that's separate).